fil_fungible_token/src/runtime/messaging.rs

```diff
@@ -32,12 +35,18 @@ pub trait MethodCaller {
         token_value: &TokenAmount,
         data: &[u8],
     ) -> Result<Receipt>;
+
+    /// Resolves the given address to it's ID address form
+    ///
+    /// If an ID address for the given address doesn't exist yet, it tries to create one by sending a
+    /// zero balance to the given address.
+    fn resolve_to_id_addr(&self, address: &Address) -> Result<ActorID>;
```

Comment:
On balance I think it would be better if this only did the resolution, and the send-to-initialise was encapsulated in the token library (and testable) rather than injected by the user.

Reply:
https://github.com/helix-onchain/filecoin/pull/46/files

New structure in here allows more testing of the behaviour on uninitialized accounts